content/docs/tutorials/cloud-tutorial.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Using InfinyOn Cloud Backend
+menu: InfinyOn Cloud Tutorial
+weight: 60
+---
+
+This tutorial will walk you through setting up an InfinyOn account, and using
+Fluvio, both as a standalone command, and as part of a larger program.
+
+## Basic Setup
+
+There are 3 main steps for setting up for the Cloud: Installing the CLI,
+registering for a Cloud account, and finally linking the two together.
+
+The following sections will walk you through how to do that.
+
+### Install Fluvio CLI
+
+Download the Fluvio CLI with the following command.
+
+%copy first-line%
+```bash
+$ curl -fsS https://packages.fluvio.io/v1/install.sh | bash
+```
+
+{{< partial file="docs/path-env.html" >}}
+
+
+### Create InfinyOn Cloud Account
+
+Head over to the [InfinyOn Cloud sign up page](https://infinyon.cloud).
+
```

Comment:
I think we should leave the "Cloud Images" out as they will change soon. Furthermore, we now allow login through OAuth which is a different workflow.